### ISSUE TITLE
LF interpreted incorrectly

### ISSUE THREAD

**Issue by SlimeDog** [ViaVersion/ViaVersion#1819]
**Describe the bug, provide any errors**
Spigot 1.15.2
ServerListPlus 3.5.2-SNAPSHOT

ViaVersion 3.0.1-b312

Hover text in server-selection screen displays incorrectly.
With the new 1.16 client and ViaVersion 3.0.1-b312 on a 1.15.2 server, the hover text in this configuration 
```
Default:
  Description:
  - |-
    &r &l%server_name% %server_version%&r &6▪&3creative &6▪&3survival &6▪&3extrahard&r &f▪private&r
    &r &6⊳&r&aCityWorld &6⊳&r&aGoT &6⊳&r&aLuckyBlock &6⊳&r&aSkyBlock &6⊳&r&aWinter &6⊳&rmore&r
  Players:
    Hover:
    - |-
      &awebsite: &rblackdog.straight8.com
      &awhitelist: &renabled - members only
      &aviaversion: &renabled - %server_version%+ clients
```
now displays as
> &awebsite: &rblackdog.straight8.com &awhitelist:[LF]&renabled - members only[LF]&aviaversion: &renabled - %server_version%+ clients

that is, all on one line with [LF] instead of actual LF or CR.

The configuration file is UTF-8 Unicode, BTW.

**How can we reproduce it?**
1. Install ViaVersion 3.0.1-b312 on Spigot 1.15.2 server
2. Start MC 1.16 client
3. Hover over text on server selector

**Expected behaviour**
The text should display as three separate lines. It does so with the MC 1.15.2 client.

**Screenshots**
![Screen Shot 2020-06-23 at 8 43 55 AM](https://user-images.githubusercontent.com/17748923/85425176-b733c300-b52d-11ea-8fb4-1389304ccf03.png)

**ViaVersion Dump:**
https://dump.viaversion.com/e17c91c528377db33e23d330be153eb987d50cfc00d5c65ac307b5b9195c20b6

**Additional server info**
Do you use a proxy (eg. BungeeCord)? No.

**Comment by SlimeDog** [comment]
Confirmed fixed in VV 3.0.1-b313. Thanks.

**Comment by SlimeDog** [comment]
Broken again in VV 3.0.2-b316.
